### DISCUSSION TITLE
web-health-checker not able to install.

### DISCUSSION THREAD

**Discussion by WhineyMonkey10** [enchant97/web-portal#82] [Q&A] [answered]
ERROR: Could not find a version that satisfies the requirement web_health_checker (from versions: none)
ERROR: No matching distribution found for web_health_checker

I've tried to install it manually. No luck with googling it either.

Any suggestions?

**Comment by enchant97** [comment, marked as answer]
Hi there, are you using the Docker image or trying to install it manually? If you are installing manually Web Portal requires at least Python 3.10

**Reply by WhineyMonkey10** [reply]
Oh, thanks!
